<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogreclamation</class>
 <widget class="QMainWindow" name="dialogreclamation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>941</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget {  
    background-color:#0f2033;
    border-top: 2px solid #C2C7CB;  
} 
QTabWidget::pane { /* The tab widget frame */  
    border-top: 0px solid #C2C7CB;  
} 
 /* Style the tab using the tab sub-control. Note that 
    it reads QTabBar _not_ QTabWidget */  
QTabBar::tab {  
    background-color:#0f2033;
    border: 1px solid #4973b4;  
    min-width: 100px; 
    min-height:22px;
    padding: 0px  2px; 
    color: rgb(255, 255, 255);
    margin-left: 1px;
    font: 75 8pt &quot; Microsoft YaHei &quot;;
}  
  
QTabBar::tab:selected, QTabBar::tab:hover {  
    background-color: rgb(42,87,147);
    margin-top: 0px;
    
}  
  
QTabBar::tab:selected {  
    
    background-image: url(:/Images/Tab_Hover.png);
    
} 
  
QTabBar::tab:!selected {  
    margin-top: 0px; /* make non-selected tabs look smaller */  
}  </string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Ajouter">
     <attribute name="title">
      <string>Ajouter Réclamation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>911</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="DescriptionR_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>Déscription Réclamation:</string>
       </property>
      </widget>
      <widget class="QLabel" name="NomC">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>Nom client:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PrenomC">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>Prenom client:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_IDR">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>130</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit { color: red }
QLineEdit[readOnly=&quot;true&quot;] { color: gray }
QDialog QLineEdit { color: brown }</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>ID Réclamation:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AjouterRec">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>570</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">Ezzaouia Mohamed Amine
QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Description">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>811</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="CINC">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>CIN client:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_CIN">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit { color: red }
QLineEdit[readOnly=&quot;true&quot;] { color: gray }
QDialog QLineEdit { color: brown }</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Afficher">
     <attribute name="title">
      <string>Afficher Réclamation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-20</y>
        <width>931</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="Supprimer">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>580</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">Ezzaouia Mohamed Amine
QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>471</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>580</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">Ezzaouia Mohamed Amine
QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QTableView" name="Reclamation">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>70</y>
         <width>501</width>
         <height>501</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="DescriptionR_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>Déscription Réclamation:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Description_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>371</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ID_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>ID Réclamation:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_IDR_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_CIN_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>250</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="NomC_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>Nom client:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PrenomC_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>Prenom client:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CINC_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  color : black; }</string>
       </property>
       <property name="text">
        <string>CIN client:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="recuperer">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>580</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">Ezzaouia Mohamed Amine
QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Récuperer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loaddata">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>580</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">Ezzaouia Mohamed Amine
QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>charger données</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exporter">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">Ezzaouia Mohamed Amine
QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>exporter</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trinomASC">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>tri nom ASC</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trinomDESC">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>tri nom DESC</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Arduino</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>531</height>
       </rect>
      </property>
      <widget class="QLabel" name="ard">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>341</width>
         <height>361</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="stop">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>450</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
